libs/protocol-client/src/lib/graphql/queries.graphql

```diff
@@ -212,9 +212,19 @@ fragment GuildUserFragment on GuildUser {
   id
   createdAt
   updatedAt
-  user_id
-  guild_id
   favorite
+  user_id
+  guild {
+    id
+    createdAt
+    updatedAt
+    discord_id
+    name
+    congrats_channel
+    logo
+    contribution_reporting_channel
+    status
+  }
```

Comment:
I don't think we need all of these fields maybe just id, discord_id, name, logo, and status